comps/vectorstores/langchain/pathway/Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+FROM pathwaycom/pathway:0.13.2-slim
+
+ENV DOCKER_BUILDKIT=1
+ENV PYTHONUNBUFFERED=1
+
+RUN apt-get update && apt-get install -y \
+    poppler-utils \
+    libreoffice \
+    libmagic-dev \
+    && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /app
+
+RUN pip install -U pathway[xpack-llm]
+RUN pip install "unstructured[all-docs] >= 0.10.28, < 0.15"
+RUN pip install langchain_openai
+RUN pip install sentence_transformers
```

Comment:
Please create a `requirements.txt` file within `comps/vectorstores/langchain/pathway` folder and add the all `pip` dependencies there.
So you'll have something line this:
```
langchain_openai
pathway[xpack-llm]
sentence_transformers
unstructured[all-docs] '>=0.10.28,< 0.15'
```
and then update your Dockerfile this way:
```
COPY requirements.txt /app/
 
RUN pip install --no-cache-dir -r requirements.txt

COPY vectorstore_pathway.py /app/ 
```